code/src/object/orxObject.c

```diff
@@ -1118,6 +1118,132 @@ void orxFASTCALL orxObject_CommandApplyImpulse(orxU32 _u32ArgNumber, const orxCO
   return;
 }
 
+/** Command: AddBody
+ */
+void orxFASTCALL orxObject_CommandAddBody(orxU32 _u32ArgNumber, const orxCOMMAND_VAR *_astArgList, orxCOMMAND_VAR *_pstResult)
+{
+  orxOBJECT *pstObject;
+
+  /* Gets object */
+  pstObject = orxOBJECT(orxStructure_Get(_astArgList[0].u64Value));
+
+  /* Valid? */
+  if(pstObject != orxNULL)
+  {
+    /* Add body */
+    orxBody_CreateFromConfig(orxSTRUCTURE(pstObject), _astArgList[1].zValue);

```

Comment:
This is unfortunately not enough to link the body to the object. The ownership also needs to be set so that, once removed, the body isn't left dangling. I'll make the required changes when integrating the PR, no need to worry about it.